package.json

```diff
@@ -47,7 +47,7 @@
 	"devDependencies": {
 		"@axe-core/react": "^4.3.1",
 		"@datadog/datadog-api-client": "^1.0.0-beta.7",
-		"@hashicorp/platform-cli": "^2.1.0",
+		"@hashicorp/platform-cli": "^2.4.0-canary-2022816231816",
```

Comment:
What's the TL;DR for `2.4.0` changes?

Reply:
Bumped `eslint-config-next` to be aware of the middleware updates